packages/core/src/js/index.ts

```diff
@@ -89,6 +89,12 @@ export { Mask, Unmask } from './replay/CustomMask';
 
 export { FeedbackButton } from './feedback/FeedbackButton';
 export { FeedbackWidget } from './feedback/FeedbackWidget';
-export { showFeedbackWidget, showFeedbackButton, hideFeedbackButton } from './feedback/FeedbackWidgetManager';
+export {
+  showFeedbackWidget,
+  showFeedbackButton,
+  hideFeedbackButton,
+  showScreenshotButton,
+  hideScreenshotButton,
```

Comment:
I think for simplicity we could leave this API out and the taking of the screenshot would be only part of the auto injected widget.

Reply:
Makes sense 👍 
Updated with 1fdf12558fad0d0aa4f4c11749a77cf86af87f4d